src/handlers/help.ts

```diff
@@ -0,0 +1,26 @@
+import {
+	constructSlackMessage,
+} from '../utils';
+
+export default async () => {
+	try {
+		const blocks = constructSlackMessage(`
+        \`/shorten help\`: show this help text
+        \`/shorten list\`: list all shortened URLs
+        \`/shorten <path> <url>\`: shorten \`<url>\` to https://vhl.ink/\`<path>\`
+        \`/shorten delete <path> <key>\`: delete <path> (requires secret \`<key>\`)
```

Comment:
```suggestion
\`/shorten help\`: show this help text
\`/shorten list\`: list all shortened URLs
\`/shorten <path> <url>\`: shorten \`<url>\` to https://vhl.ink/\`<path>\`
\`/shorten delete <path> <key>\`: delete <path> (requires secret \`<key>\`)
```
The indents are also printed. The first one doesn't matter, but I think it would make sense to also unindent that one to make it fit with the other ones.